docker/2.1/Dockerfile.gpu

```diff
@@ -0,0 +1,132 @@
+FROM nvidia/cuda:10.1-base-ubuntu18.04
+
+LABEL maintainer="Amazon AI"
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+ARG PYTHON=python3
+ARG PIP=pip3
+ARG TFS_SHORT_VERSION=2.1
+ARG TFS_URL=https://tensorflow-aws.s3-us-west-2.amazonaws.com/2.1/Serving/GPU/tensorflow_model_server
+
+ENV NCCL_VERSION=2.4.7-1+cuda10.1
+ENV CUDNN_VERSION=7.6.2.24-1+cuda10.1
+ENV TF_TENSORRT_VERSION=6.0.1
+
+# See http://bugs.python.org/issue19846
+ENV LANG=C.UTF-8
+ENV PYTHONDONTWRITEBYTECODE=1
+# Python won’t try to write .pyc or .pyo files on the import of source modules
+ENV PYTHONUNBUFFERED=1
+ENV SAGEMAKER_TFS_VERSION="${TFS_SHORT_VERSION}"
+ENV PATH="$PATH:/sagemaker"
+ENV MODEL_BASE_PATH=/models
+# The only required piece is the model name in order to differentiate endpoints
+ENV MODEL_NAME=model
+# Fix for the interactive mode during an install in step 21
+ENV DEBIAN_FRONTEND=noninteractive
+
+# allow unauthenticated and allow downgrades for special libcublas library
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends --allow-unauthenticated --allow-downgrades\
+    ca-certificates \
+    cuda-command-line-tools-10-1 \
+    cuda-cufft-10-1 \
+    cuda-curand-10-1 \
+    cuda-cusolver-10-1 \
+    cuda-cusparse-10-1 \
+    #cuda-cublas-dev not available with 10-1, install libcublas instead
+    libcublas10=10.1.0.105-1 \
+    libcublas-dev=10.1.0.105-1 \
+    libcudnn7=${CUDNN_VERSION} \
+    libnccl2=${NCCL_VERSION} \
+    libgomp1 \
+    curl \
+    git \
+    wget \
+    vim \
+    build-essential \
+    zlib1g-dev \
+    python3 \
+    python3-pip \
+    python3-setuptools \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+# The 'apt-get install' of nvinfer-runtime-trt-repo-ubuntu1804-4.0.1-ga-cuda10.0
+# adds a new list which contains libnvinfer library, so it needs another
+# 'apt-get update' to retrieve that list before it can actually install the
+# library.
+# We don't install libnvinfer-dev since we don't need to build against TensorRT,
+# and libnvinfer4 doesn't contain libnvinfer.a static library.
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends nvinfer-runtime-trt-repo-ubuntu1804-5.0.2-ga-cuda10.0 \
```

Comment:
Please add justification in a comment for why this particular library is version 5.0.2 and cuda 10.0, while the rest of the libraries are for cuda 10.1.